constants.py

```diff
@@ -995,3 +995,23 @@
     -5.543945 * kMetersPerInch,
     Rotation3d(0, (90 + 51.944769) * kRadiansPerDegree, 0),
 )
+# Climber constants
+kClimberCANID = 57
+kClimberName = "ClimberMotor"
+kClimberPGain = 0.12
+kClimberIGain = 0
+kClimberDGain = 0
+
+kClimberPositionKey = "climber/position"
+kClimberStateKey = "climber/state"
+kClimberTargetKey = "climber/target"
```

Comment:
Maybe add an atPosition boolean that publishes like the intake and elevator? Just to make it easier to see